src/docs/references/availability/index.md

```diff
@@ -128,18 +132,47 @@ For time-based plans, both availability exceptions and bookings are
 interpreted literally, i.e. covering the exact time intervals determined
 by their start and end times.
 
+## Booking states
+
+A booking can be in one of several possible states: `pending`,
+`proposed`, `accepted`, `canceled` or `declined`. Bookings change state
+only through a corresponding transaction transition, using one of the
+[booking-specific actions](/references/transaction-process-actions/#bookings).
+
+All bookings in `pending` or `accepted` states count as reservation
+against the listing's availability. On the other hand, bookings in the
+`proposed`, `canceled` or `declined` states do not affect the
+availability of the listing.
+
+The figure below illustrates the possibel booking states and transitions
+and the corresponding actions that you can use in your transaction
+process.
+
+![Booking states](./booking-states.png)
```

Comment:
Comment: My small improvement idea here is to show the diagram without `:action/update-booking` transitions. They add a lot of noise but are not critical for understanding how the states work and how you transition between them. Simply mentioning that there's also ´:action/update-booking` which can be used to update times and display times but that does not affect state, would suffice in this context.

Reply:
Good idea!